sandbox/index.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+const fs = require('fs')
+
+const path = require('path')
+
+const express = require('express')
+
+const webpackDevMiddleware = require('webpack-dev-middleware')
+
+const webpack = require('webpack')
```

Comment:
Can we please get rid of the excessive newlines? Nothing against lots of line breaks, but here it's a little excessive. We can group local and external dependencies though

Reply:
Sure, will fix this! I was certain `s-c` eslint config wasn't allowing me to leave out the newlines between imports before... 🤔